Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetByStreetcodeId/GetTagByStreetcodeIdHandler.cs

```diff
@@ -11,7 +11,9 @@ public class GetTagByStreetcodeIdHandler : IRequestHandler<GetTagByStreetcodeIdQ
     private readonly IMapper _mapper;
     private readonly IRepositoryWrapper _repositoryWrapper;
 
-    public GetTagByStreetcodeIdHandler(IRepositoryWrapper repositoryWrapper, IMapper mapper)
+    public GetTagByStreetcodeIdHandler(
+        IRepositoryWrapper repositoryWrapper,
+        IMapper mapper)
```

Comment:
why did you change the view of the arguments?

Reply:
Actually, I do not remember that I have changed it...